test/system/065-cp.bats

```diff
@@ -68,6 +72,38 @@ load helpers
        "copy into nonexistent path in container"
 
     run_podman rm -f cpcontainer
+
+    # CREATED container
+    while read id dest dest_fullname description; do
+        run_podman create --name cpcontainer --workdir=/srv $cpimage cat $dest_fullname
+        run_podman cp $srcdir/hostfile$id cpcontainer:$dest
+        run_podman start -a cpcontainer
+        is "$output" "${randomcontent[$id]}" "$description (cp -> ctr:$dest)"
+        run_podman rm -f cpcontainer
+    done < <(parse_table "$tests")
+
+    run_podman rmi -f $cpimage
+}
+
+@test "podman cp file from host to container tmpfs mount" {
+    srcdir=$PODMAN_TMPDIR/cp-test-file-host-to-ctr
+    mkdir -p $srcdir
+    content=$(random_string 20)
+    echo $content > $srcdir/file
+
+    # RUNNING container
+    run_podman run -d --mount type=tmpfs,dst=/tmp --name cpcontainer $IMAGE sleep infinity
+    run_podman cp $srcdir/file cpcontainer:/tmp
+    run_podman exec cpcontainer cat /tmp/file
+    is "$output" "${content}" "cp to container's tmpfs"
```

Comment:
A lot of people—myself first among those—are simple monkeys who, when we see an error message, grep directly for it in the test sources to find the relevant place. When error messages are identical in multiple places, brainless monkeys like me get confused.

Would you consider changing this to "cp to running container's tmpfs", and line 105 to "cp to created container's tmpfs"?

Reply:
Sure!